src/main/java/org/terasology/module/inventory/events/RequestInventoryEvent.java

```diff
@@ -3,8 +3,13 @@
 package org.terasology.module.inventory.events;
 
 import org.terasology.engine.entitySystem.event.Event;
+import org.terasology.module.inventory.components.InventoryItemComponent;
+
+import java.util.List;
 
-/**
- */
 public class RequestInventoryEvent implements Event {
```

Comment:
I'd appreciate if you could add documentation for this event similar to the [events in Health](https://github.com/Terasology/Health/tree/develop/src/main/java/org/terasology/module/health/events). The explanation of [Event Types](https://github.com/MovingBlocks/Terasology/wiki/Event-Types) might be helpful.

Reply:
I'll approve an merge this PR, but we should do another iteration over the docstring.